microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Simulate PWA Window Controls Overlay
+description: Simulate PWA Window Controls Overlay.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 05/13/2024
+---
+# Simulate PWA Window Controls Overlay
+
+In DevTools, in the **Application** tool, you can simulate the Window Controls Overlay feature that Progressive Web Apps (PWAs) have, without having to install the PWA.  The **Application** tool has a **Window Controls Overlay** section under **Manifest**, to display a preview of how your Progressive Web App (PWA) handles window controls, without having to install the PWA and make the styling changes that are necessary to accommodate window controls:
+
+![The 'Window Overlay Controls' section in the Application tool](./simulate-window-controls-overlay-images/pwa-window-controls.png)
+
+1. In the manifest file, set the `display-override` field to `window-controls-overlay`:
+
+   ```json
+   {
+     "display_override": ["window-controls-overlay"]
+   }
+   ```
+
+   For more information, see [Enable the Window Controls Overlay in your app](../../progressive-web-apps-chromium/how-to/window-controls-overlay.md#enable-the-window-controls-overlay-in-your-app) in _Display content in the title bar_.
+
+1. In Microsoft Edge, select **Settings and more** > **More tools** > **Developer tools**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
+
+1. On the **Activity Bar**, select  the **Application** tab.  If that tab isn't visible, click the **More tools** (![More tools icon](./simulate-window-controls-overlay-images/more-tools-icon.png)) button.
+
+1. In the **Application** tool, expand **Manifest** in the upper left, and then select **Window Controls Overlay**.
+
+1. In the **Window Controls Overlay** section on the right, select the checkbox **Emulate the Window Controls Overlay on** \<platform\>, and select a platform, such as Windows.
+
+<!-- todo: then what?  what action, what result?
+1. 
+-->
+
+Use the Window Controls Overlay to have control over the title bar area that's currently displayed in standalone display mode.  Window Controls Overlay (WCO) is a set of features that work together to provide just the essential controls needed for the app window.  This layout frees up more space for the web content layer.
+<!-- from https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa#window-controls-overlay-origin-trials -->
+
+Use the **Window Controls Overlay** section to emulate window controls for Windows, Linux, and macOS.
```

Comment:
This belongs in a section earlier in the doc. Maybe a new H2 section after the intro? How about we structure the article like this:

```
# Simulate PWA Window Controls Overlay

Intro "In DevTools, in the **Application** tool, you can simulate the Window Controls Overlay (WCO) feature that Progressive Web Apps (PWAs)..."

## Using the WCO feature in your PWA

Quick reminder of what WCO is, and link to https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay

## Simulate WCO in DevTools

Your list of steps + screenshots.